references/oidc-mock/apiproxy/policies/AM-SetDiscoveryDocument.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ Copyright 2020 Google LLC
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<AssignMessage async="false" continueOnError="false" enabled="true" name="AM-SetDiscoveryDocument">
+    <DisplayName>AM-SetDiscoveryDocument</DisplayName>
+    <Properties/>
+    <Set>
+        <Payload contentType="application/json">{
+  "issuer": "https://{organization.name}-{environment.name}.apigee.net/v1/openid-connect",
+  "authorization_endpoint": "https://{organization.name}-{environment.name}.apigee.net/v1/openid-connect/auth",
+  "token_endpoint": "https://{organization.name}-{environment.name}.apigee.net/v1/openid-connect/token",
+  "introspection_endpoint": "https://{organization.name}-{environment.name}.apigee.net/v1/openid-connect/token/introspect",
+  "userinfo_endpoint": "https://{organization.name}-{environment.name}.apigee.net/v1/openid-connect/userinfo",
+  "end_session_endpoint": "https://{organization.name}-{environment.name}.apigee.net/v1/openid-connect/logout",
+  "jwks_uri": "https://{organization.name}-{environment.name}.apigee.net/v1/openid-connect/certs",
+  "check_session_iframe": "https://{organization.name}-{environment.name}.apigee.net/v1/openid-connect/login-status-iframe.html",
+  "grant_types_supported": [
```

Comment:
we should reduce this to the list of grant types actually supported by the mock.

Today that is only `authorization_code` but we may extend it in future